server/conf/i18n/messages

```diff
@@ -326,6 +326,46 @@ toast.mayNotQualify=Based on your responses, you may not qualify for the {0}. If
 toast.mayNotQualifyTi=Based on your responses, your client may not qualify for the {0}. If your client''s information has changed you can edit your answers and then proceed with the application.
 banner.errorSavingApplication=Error saving application
 
+# Alert title when applicant may be eligible for a program
+alert.eligibility_applicant_eligible_title=You may be eligible for this program
+# Alert text when applicant may be eligible for a program
+alert.eligibility_applicant_eligible_text=Based on your responses, you are likely eligible for this program.
+
+# Alert title when applicant may not be eligible for a program
+alert.eligibility_applicant_not_eligible_title=You may not be eligible for this program
+# Alert text when applicant may not be eligible for a program
+alert.eligibility_applicant_not_eligible_text=Based on your responses, you may not be eligible for this program. If your information has changed, you can edit your answer and proceed with the application.
+
+# Alert title when applicant may be eligible for a program after the application gets updated to using a newer program version
+alert.eligibility_applicant_fastforwarded_eligible_title=You may be eligible for this program
+# Alert text when applicant may be eligible for a program after the application gets updated to using a newer program version
+alert.eligibility_applicant_fastforwarded_eligible_text=There may be new or updated questions that affect your answers or eligibility. Please review all questions below.
+
+# Alert title when applicant may not be eligible for a program after the application gets updated to using a newer program version
+alert.eligibility_applicant_fastforwarded_not_eligible_title=You may not be eligible for this program anymore
```

Comment:
**suggestion:** Same with this one about having the same text as `alert.eligibility_applicant_not_eligible_title`